src/pages/upload-page/components/FinishPage.module.css

```diff
@@ -10,14 +10,15 @@ sbuttons {
 }
 
 .continueButton {
+  all: unset;
```

Comment:
@milindNshah so from what i can tell, this resets a button to look like a div. will get some screenshots to confirm as well